napari/layers/_base_layer/model.py

```diff
@@ -226,15 +234,17 @@ def _after_set_viewer(self, prev):
         if self.viewer is not None:
             self.refresh()
 
-    def _set_view_slice(self, indices):
-        """Called whenever the sliders change. Sets the current view given a
-        specific slice to view.
+    @abstractmethod
```

Comment:
Why is this an abstract method? Does every layer type need to respond to view slices?

Reply:
Maybe I am confusing the exact semantics of 'abstract method' in python. We don't need every layer to explicitly define such methods.  The base class can provide a default implementation that does nothing and derived classes can decide whether they want to respond and react to these calls or not...